README.md

```diff
@@ -766,7 +766,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 **[⬆️ Back to Top](#table-of-contents)**
 
 ## Translation Management
-
+  * [VideoDubber.ai](https://videodubber.ai/) — Video Translation through UI and API, Subtitles Generation and Translation, free for first 30 minutes of subtitles generation and 5 minutes of video translation for 150+ languages and accents.
```

Comment:
sorry, 30 minutes free isnt useful, to quality for this list the free tier need to be actually useful for developers to do their private work and integration with APIs etc without cost.

Reply:
I aggree with you, @ripienaar 
**The Free trial features of the platform is very limited** -
"_In the free trial only 40 secs of watermark-free video is translated.
To unlock the full video and other advanced features like subtitle editing and voice cloning keeping your original voice, subscribe to the pro version."_ 

![Screenshot from 2023-12-20 19-07-21](https://github.com/ripienaar/free-for-dev/assets/97863234/ee227010-361b-4329-be92-f9ae15df694a)

Reply:
Thanks yeah this is not a free tier it’s a gimmick. Closed it thanks.

Reply:
> sorry, 30 minutes free isnt useful, to quality for this list the free tier need to be actually useful for developers to do their private work and integration with APIs etc without cost.

Hi @ripienaar , I am a solopreneur and will need your backlink/traffic.
**I am willing to modify the free tier according to your suggestion keeping the saas profitable atleast.**
Can you suggest some limits which you think will be useful for devs?
The 40 sec limit was there so that anyone can generate subtitles and translation for their short videos. 
But if you suggest we are ready to increase both the limits within reasonable range of profitability.

Reply:
Sorry this is not a marketing business.  If your free tier is real, long term and offer significant value its welcome here.

The purpose of this list is to serve developers, not your marketing needs.

Reply:
That's fine. What is your recommended length in the free tier?

Reply:
Think about developers who want to integrate their clients offering into your service - their clients will pay for your service.

What we dont want is for developers to pay to integrate into your service? Over the course of designing a solution for a client it could mean many tests, tests that would eat through minutes.

Make an informed decision about your use case, I cannot dictate it.  I do know when something is not useful, a few minutes is a gimmik.

Sorry, cant spend more time offering you this kind of support, if its useful and you are offering a genuine free tier and not just to abuse this list for marketing we'll know it.